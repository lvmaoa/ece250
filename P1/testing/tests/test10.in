m 7
front
back
push_front "news" "news.google.com"
push_back "google" "www.google.com"
push_front "stackexchange" "www.stackexchange.com"
push_back "bored" "www.boredpanda.com"
push_back "uw" "www.uwaterloo.ca"
push_front "piazza" "www.piazza.com"
front
back
find "piazza"
find "chuckECheese"
exit

