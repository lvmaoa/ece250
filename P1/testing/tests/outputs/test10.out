success
failure
failure
success
success
success
success
success
success
front is "piazza" "www.piazza.com"
back is "uw" "www.uwaterloo.ca"
found "piazza" "www.piazza.com"
not found "chuckECheese"
